router/routes.js

```diff
@@ -1,219 +0,0 @@
-const express = require("express") ;
-const { createemployee, getAllUser, getSingleUser, update, deleteUser } = require("../controller/employeeController");
-const router  =express.Router() ;
-const companytable = require("../models").company;
-
-// api for creating new user details  
-router.post("/create" ,createemployee)
-  
-
-// api for getting all user details 
-router.get("/alluser",getAllUser)
-
-
-// api for getting a single user by id 
-router.get("/singleUser/:id",getSingleUser)
-
-// api for updating the user details 
-router.put("/update/:id", update)
-
-
-
-// api for deleting user from database 
-router.delete("/delete/:id", deleteUser);
-
-
-router.get("/" ,(req,res) => {
-    res.json({
-        status: true ,
-        message : "Welcome to page "
-    })
-})
-
-module.exports = router ;
-
-// const express = require("express") ;
-// const router  =express.Router() ;
```

Comment:
remove commented out code before giving PR for review

Reply:
ok